pkg/goods/support/host-support-bundle.tmpl.yaml

```
@@ -153,6 +153,9 @@ spec:
       collectorName: 'check-umount'
       command: 'sh'
       args: ['-c', 'command -v umount']
+  - copy:
+      collectorName: installer/lam-service-config
+      path: /etc/systemd/system/local-artifact-mirror.service.d/embedded-cluster.conf
```

does this support globs? is this possible?

```suggestion
      path: /etc/systemd/system/local-artifact-mirror.service.d/*
```

---

I think just a `/` is enough, no need for the `/*`

---

https://troubleshoot.sh/docs/host-collect-analyze/copy/ `*` is in the logs though, so yeah let's go with that